### PR TITLE
Add missing property DOMNamedNodeMap::$length

### PR DESCRIPTION
**Pull request by ondrejmirtes**
https://www.php.net/manual/en/class.domnamednodemap.php#domnamednodemap.props.length

### CODE REVIEW

File: dom/dom_c.php [hunk 1]
```diff
@@ -1080,6 +1080,14 @@ public function count() {}
...
 class DOMNamedNodeMap implements Traversable, Countable {
 
+    /**
+     * @var int
+     * The number of nodes in the map. The range of valid child node indices is 0 to length - 1 inclusive.
+     * @link https://www.php.net/manual/en/class.domnamednodemap.php#domnamednodemap.props.length
+     */
+    public $length;
+	
+
```
Comment (schlessera) [inline dom/dom_c.php:1090]: Travis CI is failing. I assume this is because of the double blank lines here?

Comment (schlessera) [inline dom/dom_c.php:1090]: There's also tabs in there, it seems (difficult to tell on GH...).
```suggestion

```

Comment (ondrejmirtes) [reply dom/dom_c.php]: Thank you. I commited the suggestion. Feel free to squash it before mege, thank you :)

**Comment by isfedorov** [comment]
Hi @ondrejmirtes 
this property is already declared in phpdoc and can be resolved successfully
Screenshot 2020-06-17 at 16 04 13
Screenshot 2020-06-17 at 15 49 16

**Comment by b1rdex** [comment]
> this property is already declared in phpdoc and can be resolved successfully
NB: there is no type in the property declaration.

**Comment by isfedorov** [comment]
>NB: there is no type in the property declaration.

@b1rdex fixed in scope of https://github.com/JetBrains/phpstorm-stubs/commit/4b340f6bc6c322b450cc3246b29dde2a4bed7783

**Pull request closed without merging**